### PR TITLE
Add golang Jenkins jnlp agent

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: golang/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM jenkins/inbound-agent:alpine as jnlp
+
+FROM golang:1.15-alpine3.12
+
+RUN apk -U add openjdk8-jre
```
Comment (slide) [inline golang/Dockerfile:5]: Any reason to use 8 here? I'd recommend using 11 if possible.

Comment (LinuxSuRen) [reply golang/Dockerfile:5]: No particular reason. I think Java 11 is ok.

Comment (LinuxSuRen) [reply golang/Dockerfile:5]: By the way how about we update other agent images from java8 to java11? What do you think?

**Review by slide — APPROVED** [review]

**Comment by LinuxSuRen** [comment]
hi @slide , I was wondering if you know when I can see the docker image from https://hub.docker.com/u/jenkins?

**Comment by oleg-nenashev** [comment]
@LinuxSuRen From what I see Trusted CI did not pick up this image. Checking why

**Comment by oleg-nenashev** [comment]
NVM, the last build has actually completed before the build was started. Got confused by timezones. The new build is running, ETA is in 30min or so

image

**Comment by LinuxSuRen** [comment]
Thank you @oleg-nenashev and @slide . The image is ready https://hub.docker.com/r/jenkins/jnlp-agent-golang.

**Pull request merged**